experimental/whisper/speedup.ipynb

```diff
@@ -666,6 +666,9 @@
     }
    ],
    "source": [
+    "# uncomment to use vanilla torch.compile instead of Kernl\n",
```

Comment:
maybe add a precision to tell to comment the optimize_model line, because if we just uncomment the line it will not do what is says